_posts/2018-10-31-cockpit-181.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Cockpit 181
+author: mvo
+date: 2018-10-31
+tags: cockpit linux
+slug: cockpit-181
+category: release
+summary: ...
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We
+release regularly.  Here are the release notes from version 181.
+
+### Graph improvements
+
+The appearance of memory and CPU graphs has been improved, and their
+colors are more accessible.
+
+![New memory graph](/images/memory-graph.png)
+
+Memory on Linux is often hard to understand and usually presented
+incorrectly in most tools. There are so many different terms, most of
+which do not matter to anyone who's not a kernel developer.
+
+In addition, Cockpit's old memory graphs were unfortunately calculated
+incorrectly as well.
+
+The new presentation of memory in Cockpit is aligned with the way
+people think about how RAM is used. It now shows:
+
+1. the amount of RAM actively used by software
+2. the amount of RAM used for cache
+3. unused RAM
+4. swap
+
+On Cockpit's overview, the memory graph displays only the value of
+active RAM and has a separate, small area to indicate swap usage. The
+detailed memory view also shows cache.
+
+Why do we not show cache on the overview and de-emphaize it on the
+detailed memory view? Linux stores often-used files in memory as cache
+to speed up performance. As cache is quickly reclaimed when needed,
+it's effectively available RAM. That's why Cockpit shows it a free
+space, as Linux considers it free too.
+
+Additional information on can be found on the website [Linux Ate My
+RAM](https://www.linuxatemyram.com/).
+
+###  Edit network interfaces of virtual machines
+
+The configuration of the network interfaces of a virtual machine can
+now be changed in Cockpit.
+
+![nic dialog](/images/nic-edit.png)
+
+### Update look of lists to match Patternfly
+
+...
+
+### Followup fixes related to the switch away from react-lite
```

Comment:
Traditionally we don't document bug fixes; we always have so many of them. I'd just leave this out.

Reply:
Does switching to React from React-lite bring any user-facing updates? If not (and I think that's the case here), I agree.

Reply:
Removed